gr-fec/lib/polar_common.cc

```diff
@@ -108,14 +108,13 @@ namespace gr {
         d_volk_temp = (unsigned char*) volk_malloc(sizeof(unsigned char) * block_size(), volk_get_alignment());
         d_volk_frozen_bit_mask = (unsigned char*) volk_malloc(sizeof(unsigned char) * block_size(), volk_get_alignment());
         d_volk_frozen_bits = (unsigned char*) volk_malloc(sizeof(unsigned char) * nfrozen, volk_get_alignment());
-        for(int i = 0; i < nfrozen; i++){
-          d_volk_frozen_bits[i] = d_frozen_bit_values[i];
-        }
+        std::copy(d_frozen_bit_values.begin(), d_frozen_bit_values.end(), d_volk_frozen_bits);
```

Comment:
It takes a while to figure out, why these lines are a good idea. This just asks for a comment. `d_frozen_bit_values` might be an empty std::vector. Fill it with default values ('0').

Reply:
Then we have a bug somewhere else! Our code shouldn't be dealing with more frozen bits than we have, right?